### ISSUE TITLE
Impact not showing in versions

### ISSUE THREAD

**Issue by galexqt** [ImpactDevelopment/ImpactIssues#3428]
I download Impact, it says installed, but wont show up in my installations.

1. download impact installer
2. click 1.12 with no optifine
3. click install
4. it says it is successfully installed
5. I go go the minecraft launcher and its not there.

I expected it to show up near the bottom left.

![image](https://user-images.githubusercontent.com/72717908/179384738-4b4026c0-4e2c-4adf-ad20-f5bc1d58583b.png)
![image](https://user-images.githubusercontent.com/72717908/179384748-07442dbe-addc-4a09-bd64-768a194bbfec.png)
![image](https://user-images.githubusercontent.com/72717908/179384766-a4f4fccf-f126-4fa7-b159-1f9de63bf889.png)
![image](https://user-images.githubusercontent.com/72717908/179384773-afee3833-ea69-47d1-bea7-007392389dae.png)
![image](https://user-images.githubusercontent.com/72717908/179384789-545e365f-5423-4ecc-9239-bc14ba5af0bf.png)

Basically, it says impact successfully installed but it doesnt show up in installations or versions of the minecraft launcher. but it does show that the impact file is in the versions so idk what to do

 

- Impact Version: 
- Using Optifine? 
- Using LiteLoader? 
- Enabled mods: 
- Operating System:

**Comment by galexqt** [comment]
The stuff at the bottom didn't show up, I'll just write them here:
Impact Version: 4.9.1 for 1.12.2
Using Optifine?: no
Using LiteLoader?: no
Enabled mods: none, deleted them all
Operating System: Windows 10

**Comment by feliscatusmeows** [comment]
Close minecraft launcher before installing imprat

**Comment by feliscatusmeows** [comment]
*impcat

**Comment by feliscatusmeows** [comment]
impact*

**Comment by CesiumCs** [comment]
im gonna consider this solved cause that just seems like the right answer to the problem
